client/src/components/EstimatedCost.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
+import form from './Form.css';
 
 const EstimatedCost = (props) => (
-  <div> Estimated Cost
-    <span id="estimated-cost-calc"> {props.estimation}</span>
+  <div className={form.estimatedPriceContainer}> 
+    <div>Estimated Cost
```

Comment:
You might want to wrap a span tag for 'Estimated Cost' maybe?

Reply:
I'll look into it